app/init.go

```diff
@@ -67,7 +68,7 @@ func loadSettings() (Settings, error) {
 }
 
 func loadDB(DB dbConnectionInfo) (*sql.DB, error) {
-	connection := fmt.Sprintf("%s:%s@/%s?charset=latin1", DB.User, DB.Pass, DB.Name)
+	connection := fmt.Sprintf("%s:%s@%s/%s?charset=latin1", DB.User, DB.Pass, DB.HostName, DB.Name)
```

Comment:
I think you need a protocol, this is my reading from https://github.com/go-sql-driver/mysql#examples

Given that in production we use a UNIX socket I'd suggest to extend the `DB` struct with a protocol field so we can configure it to `unix`